source/tool/tests/software/test_util.py

```diff
@@ -133,6 +133,158 @@ def read_phys_mem(self, pa_hi, pa_lo, length):
         self._chipsec_util("acpi list", ACPIHelper)
         self.assertIn("ABCD: 0x0000000000000300", self.log)
 
+    def test_acpi_facp_list(self):
+
+        class ACPIHelper(mock_helper.TestHelper):
```

Comment:
This class should be moved to mock_helper. You can then modify part of it for each specific test. (similar to https://github.com/tweksteen/chipsec/pull/6/commits/276b1fb620784afc85b0bbaf18628c7dc82cf3a0)